values/cassandra-external/values.yaml

```diff
@@ -0,0 +1,4 @@
+IPs:
+  - 10.133.0.4
+  - 10.133.0.13
+  - 10.133.0.15
```

Comment:
issue: I think, is a leftover from bundling or test runs. Same goes for ES and minio below.

Reply:
Thanks. I commited this by accident